backend/src/main/java/meowhub/backend/jpa_buddy/UserToken.java

```diff
@@ -0,0 +1,62 @@
+package meowhub.backend.jpa_buddy;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Size;
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.ColumnDefault;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import java.time.LocalDate;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "USER_TOKENS", schema = "mh_users")
+public class UserToken {
+    @Id
+    @Size(max = 36)
+    @ColumnDefault("sys_guid()")
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @Column(name = "ID", nullable = false, length = 36)
+    private String id;
+
+    @NotNull
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @OnDelete(action = OnDeleteAction.RESTRICT)
+    @JoinColumn(name = "USER_ID", nullable = false)
+    private User user;
+
+    @Size(max = 512)
+    @Column(name = "TOKEN", length = 512)
+    private String token;
+
```

Comment:
i modified the size of the column that will store tokens in the future. Currently, our token only stores user basic data and roles, and it will be more than enough. Token size monitoring is recommended in the future